PyDevRemoteDebug/PyDevRemoteDebug.py

```diff
@@ -448,8 +473,8 @@ def connect(self):
 
     # Connect to the debugger
     try:
-      pydevd.settrace('localhost', port=self.portNumber, stdoutToServer=True, stderrToServer=True, suspend=False)
-    except Exception, e:
+      pydevd.settrace('localhost', port=port, stdoutToServer=True, stderrToServer=True, suspend=False)
```

Comment:
port=port may be valid but looks a bit strange. I think this would be more clear:
port = portNumber